creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -76,15 +130,28 @@ class RehydrateIpfsQueue {
    * @param {object} logContext
    */
   async addRehydrateIpfsFromFsIfNecessaryTask (multihash, storagePath, { logContext }, filename = null) {
+    const taskType = 'rehydrateIpfsFromFsIfNecessary'
+
     if (enableRehydrate) {
-      this.logStatus(logContext, 'Adding a rehydrateIpfsFromFsIfNecessary task to the queue!')
+      this.logStatus(logContext, `Attempting to add a ${taskType} task to the queue! CID: ${multihash}`)
+
+      // disable adding to queue if max queue count is greater than some threshold
       const count = await this.queue.count()
```

Comment:
not relevant to this PR, but maybe we should add a log here to indicate that the queue size has reached its limit

Reply:
so i modified the logs to be more specific instead of just print out the generic "Adding a rehydrateIpfsFromFsIfNecessary task to the queue!"